src/ucs/sys/sys.c

```diff
@@ -340,6 +355,11 @@ size_t ucs_get_page_size()
 
     if (page_size == 0) {
         page_size = ucs_sysconf(_SC_PAGESIZE);
+        if (page_size < 0) {
```

Comment:
page_size should be changed to 'long' so it could be <0

Reply:
I'm surprised testing didn't catch this ... neither compiler